### DISCUSSION TITLE
Skip Fragmented List when navigating

### DISCUSSION THREAD

**Discussion by simasch** [marp-team/marp#365] [Ideas] [answered]
When navigating, fragmented lists become unhandy because, for every bullet point, I have to the key. 
Especially when going back to the previous slide, this is painful

There should be a key that switches the slides and skips the fragmented lists.

**Comment by yhatt** [comment, marked as answer]
Try <kbd>Shift</kbd> + arrow key.

- Related
  - https://github.com/marp-team/marp-cli/issues/206
  - https://github.com/marp-team/marp-cli/pull/248

**Comment by simasch** [comment]
Hi @yhatt 

Amazing! Thanks. I couldn't find any documentation about navigation. Did I miss it?

**Reply by yhatt** [reply]
Surely this key combination is not documented. Pull request to Marp CLI is welcome.

https://github.com/marp-team/marp-cli/blob/7e58d83b12d4b8dc0c738e539378e3993bbe85f0/README.md?plain=1#L282

**Comment by simasch** [comment]
@yhatt Would this be the right place? I was more looking for a user documentation on https://marp.app/

**Reply by yhatt** [reply]
For the present. There is a detailed documentation in README.md for each Marp tools instead of marp.app.
